data/json/items/chemicals_and_resources.json

```diff
@@ -76,7 +76,7 @@
     "name_plural" : "limestone",
     "symbol" : "=",
     "color" : "dark_gray",
-    "description" : "A handful of small chunks of limestone.  Useful for experiments at science fairs, but pretty useless as ammo.  Maybe something could be crafted from this stuff…",
+    "description" : "A handful of small chunks of limestone.  Useful for experiments at science fairs, but pretty useless as ammo.  Maybe something could be crafted from this stuff...",
```

Comment:
How is proper ellipsis character as opposed to three periods a "typo"?
Ellipsis already used for [shortening strings to fit UI](https://github.com/CleverRaven/Cataclysm-DDA/blob/419dca391d1f2a8c2f953bfa084454ee15901594/src/catacharset.cpp#L616).

Reply:
In other description/mission text, Ellipsis is replaced with three commas instead.

Also ellipsis is an ambiguous-width characters, which will cause issue like #12474.